src/pages/signup/StepThree.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import Button from '@mui/material/Button';
+
+
+export default function StepThree() {
+  return (
+    <div>
+         <FormControl fullWidth>
+            <InputLabel>Gender</InputLabel>
+            <Select
+                labelId="demo-simple-select-label"
+                id="demo-simple-select"
+                value={gender}
+                label="Gender"
+                onChange={handleChange}
+            >
+                <MenuItem value={male}>Male</MenuItem>
+                <MenuItem value={female}>Female</MenuItem>
+                <MenuItem value={other}>Other</MenuItem>
```

Comment:
Here the values you supplied for the `value` prop are variables that aren't defined. I think they should be string literals 
```suggestion
                <MenuItem value={"male"}>Male</MenuItem>
                <MenuItem value={"female"}>Female</MenuItem>
                <MenuItem value={"other"}>Other</MenuItem>
```